package.json

```diff
@@ -103,7 +103,7 @@
     "@babel/core": "^7.3.4",
     "@babel/polyfill": "^7.2.5",
     "@babel/register": "^7.0.0",
-    "@elastic/charts": "^3.11.2",
+    "@elastic/charts": "^4.2.1",
```

Comment:
This change shouldn't be here - elastic-charts version 4 is already in `feature/lens`: https://github.com/elastic/kibana/blob/feature/lens/package.json#L105

Probably some GH foo